scala/lambdas/ingest-flow-control/src/main/scala/uk/gov/nationalarchives/ingestflowcontrol/Lambda.scala

```diff
@@ -0,0 +1,203 @@
+package uk.gov.nationalarchives.ingestflowcontrol
+
+import cats.effect.*
+import io.circe.generic.auto.*
+import pureconfig.ConfigReader
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+import uk.gov.nationalarchives.DADynamoDBClient.DADynamoDbWriteItemRequest
+import uk.gov.nationalarchives.DASFNClient.Status.Running
+import uk.gov.nationalarchives.ingestflowcontrol.Lambda.*
+import uk.gov.nationalarchives.utils.{Generators, LambdaRunner}
+import uk.gov.nationalarchives.{DADynamoDBClient, DASFNClient, DASSMClient}
+import uk.gov.nationalarchives.dynamoformatters.DynamoFormatters.{*, given}
+
+import java.time.Instant
+import scala.annotation.tailrec
+
+class Lambda extends LambdaRunner[Input, Unit, Config, Dependencies] {
+
+  override def handler: (Input, Config, Dependencies) => IO[Unit] = { (input, config, dependencies) =>
```

Comment:
You don't need this outer brace (`IO[Unit] = {`) since you are enclosing the code with a brace on line 20